index.js

```diff
@@ -177,16 +176,16 @@ Parser.prototype._handleError = function _handleError(line) {
 
       // Need to set this value
       if (m[0] === 'actual') {
-        lastAssert.error.actual = trim(m[1]);
+        lastAssert.error.actual = m[1].trim();
       }
     }
 
     // outputting expected/actual object or array
     if (this.currentNextLineError) {
-      lastAssert.error[this.currentNextLineError] = trim(line);
+      lastAssert.error[this.currentNextLineError] = line.trim();
       this.currentNextLineError = null;
     }
-    else if (trim(m[1]) === '|-') {
+    else if ((m[1] || '').trim() === '|-') {
```

Comment:
like those cases can be replaced
```suggestion
    else if (m[1]?.trim() === '|-') {
```